quickstarts/maven/hello-world/pom.xml

```diff
@@ -72,7 +72,7 @@
         <configuration>
           <images>
             <image>
-              <name>helloworld-java:${project.version}</name>
+              <name>waynemorphic/helloworld-java:${project.version}</name>
```

Comment:
I don't think it's good idea to commit your personal dockerhub username in a project's quickstart.

Reply:
Okay, noted. Is this correct though? 
> _Tentative opinion ->_ What happens under the hood is, k8s pulls the built image from docker hub. However, since `hello-world...` image does not exist in docker hub, the pull fails, thus we experience `ImagePullErr` followed by `CrashLoopBackOff` as k8s tries to pull the image multiple times.

Reply:
Yes, this is the default behavior if image is not present in local docker host. However, I'm not sure against which docker host you're running `mvn k8s:build`. If you build image, it should be build inside local docker host. 

Are you testing this with minikube or kind?

Reply:
Yes, I am testing with minikube in docker desktop

Reply:
@waynemorphic : Is the behavior same when you do it like this?
```
eval $(minikube -p minikube docker-env)
mvn k8s:build k8s:resource k8s:apply
```

Reply:
`CrashLoopBackOff` persisted when I did that

Reply:
@waynemorphic : Are you on Windows?

Reply:
on macOS, ventura

Reply:
Umm, strange. Let me try out your PR later today on my machine.

Reply:
cool

Reply:
You would also need to add configuration for exposing ports now that you're deploying a web application:
```
        <configuration>
          <images>
            <image>
              <name>waynemorphic/helloworld-java:${project.version}</name>
              <alias>hello-world</alias>
              <build>
                <from>openjdk:latest</from>
                <cmd>java -jar maven/${project.artifactId}-${project.version}.jar</cmd>
                <ports>
                  <port>8081</port>
                </ports>
              </build>
            </image>
          </images>
        </configuration>
```